rt_dashboard/web.py

```diff
@@ -256,8 +259,14 @@ def list_jobs(queue_name, page):
             )
         )
 
-        offset = (current_page - 1) * per_page
-        jobs = [serialize_job(job) for job in queue.get_tasks(offset, per_page)]
+        if reverse_order:
+            start = -1 - (current_page - 1) * per_page
+            end = start - per_page
+            jobs = reversed(queue.get_tasks(end, start))
+        else:
+            offset = (current_page - 1) * per_page
+            jobs = queue.get_tasks(offset, per_page)
```

Comment:
I'm not sure I understand why `per_page` is the second argument. If `current_page` is 2, then we call `queue.get_tasks(per_page, per_page)`, is that correct?

Reply:
Or wait, it is the `length` argument, which explains it. Let me think again about the reverse order :)